synapse/lib/aha.py

```diff
@@ -257,6 +262,41 @@ async def signUserCsr(self, csrtext, signas=None):
         await self._reqUserAllowed(('aha', 'csr', 'user'))
         return await self.cell.signUserCsr(csrtext, signas=signas)
 
+    @s_cell.adminapi()
+    async def addAhaPool(self, name, info):
+        return await self.cell.addAhaPool(name, info)
+
+    @s_cell.adminapi()
+    async def delAhaPool(self, name):
+        return await self.cell.delAhaPool(name)
+
+    @s_cell.adminapi()
+    async def addAhaPoolSvc(self, poolname, svcname, info):
+        return await self.cell.addAhaPoolSvc(poolname, svcname, info)
+
+    @s_cell.adminapi()
+    async def delAhaPoolSvc(self, poolname, svcname):
+        return await self.cell.delAhaPoolSvc(poolname, svcname)
+
+    async def iterPoolTopo(self, name):
+
+        username = self.user.name.split('@')[0]
```

Comment:
A user which is not allowed to get a given network is allowed to get the topology related to that network.